components/shortcut/shortcut.app.js

```diff
@@ -0,0 +1,220 @@
+     * @params {integer} requestedById - The ID of the member that requested the story.
+     * @params {Date} startedAtOverride - Manual override for the time/date the Story was started.
+     * @params {array} storyLinks - An array of story links attached to the story. Each story link
+     * must have the
+     * [CreateStoryLinkParams](https://shortcut.com/api/rest/v3#CreateStoryLinkParams) structure.
+     * @params {string} storyType - The type of story (feature, bug, chore).
+     * @params {array} tasks - An array of tasks connected to the story. Each task must have the
+     * [CreateTaskParams](https://shortcut.com/api/rest/v3#CreateTaskParams) structure.
+     * @params {string} updatedAt - The time/date the story was updated.
+     * @params {integer} workflowStateId - The ID of the workflow state the story will be in.
+     * @returns {story: object } An object with the created story, as per input provided, default
+     * values. See the full schema at
+     * [Create Story Responses](https://shortcut.com/api/rest/v3#Responses-79271).
+     */
+    async createStory(data) {
+      return await this.api().createStory(data);
+    },
+    /**
+     * Returns a list of all Epics and their attributes.
+     * @returns {epics: array } An array of all epics in the connected Shortcut account.
+     * See the [Epic schema](https://shortcut.com/api/rest/v3#Epic) at the API docs.
+     */
+    async listEpics() {
+      return await this.api().listEpics();
+    },
+    /**
+     * Returns a list of all Files.
+     * @returns {files: array } An array of all files in the connected Shortcut
+     * account.
+     * See the [Files schema](https://shortcut.com/api/rest/v3#List-Files) at the API docs.
+     */
+    async listFiles() {
+      return await this.api().listFiles();
+    },
+    /**
+     * Returns a list of all Linked Files.
+     * @returns {linkedFiles: array } An array of all linked files in the connected shortcut
+     * account.
+     * See the [Linked File schema](https://shortcut.com/api/rest/v3#LinkedFile) at the API docs.
+     */
+    async listLinkedFiles() {
+      return await this.api().listLinkedFiles();
+    },
+    /**
+     * Returns a list of all Members and their attributes.
+     * @returns {members: array } An array of all members in the connected Shortcut account.
+     * See the [Members schema](https://shortcut.com/api/rest/v3#Member) at the API docs.
+     */
+    async listMembers() {
+      return await this.api().listMembers();
+    },
+    /**
+     * Returns a list of all Members as options to use as dynamically populated prop's options.
+     * @returns {members: array } An array where each element has the `value` and `label`
+     * properties, to be used as dynamically populated prop's options. `value` is set to the `id`
+     * of the shortcut member, and `label` to the member's name.
+     */
+    async listMembersAsOptions() {
+      const members = await this.callWithRetry("listMembers");
```

Comment:
In an earlier review comment by Jonathon he suggested both ways, the one I implemented, and the one you are suggesting. Can you please advice if you definetely want this change? Just trying to save some effort here, which otherwise has been used.

Reply:
Yeah no problem I saw that previous review, it was just a suggestion, I totally understand if you want to leave it as it is right now! 👍